lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -314,6 +314,16 @@ def setup_purchase(money, options = {})
         commit(:post, 'payment_intents', post, options)
       end
 
+      def inquire(intent_id, options = {})
```

Comment:
Can you make it be `authorization` instead of `intent_id` just to follow the same patter we have for the other gateways